packages/now-cli/scripts/build.ts

```diff
@@ -39,11 +39,14 @@ async function createBuildersTarball() {
     }
   }
 
-  const yarn = join(dirRoot, '../../node_modules/yarn/bin/yarn.js');
-  await execa(process.execPath, [yarn, 'add', '--no-lockfile', ...builders], {
-    cwd: buildersDir,
-    stdio: 'inherit',
-  });
+  await execa(
+    'npm',
+    ['install', '--save-exact', '--save-dev', '--no-package-lock', ...builders],
```

Comment:
I thought we wanted to stop bundling builders from the registry and instead use the local files from the monorepo, possibly with `npm pack`?

Reply:
That's separate from this issue, and will only apply to the core `@now` Runtimes. Community runtimes and custom URLs will still need this logic to be supported.

Reply:
Indeed. So in this case you are just replacing all instances of yarn, even though this line from `build.ts` is not actually invoked during `now dev` right?

Reply:
Right. It's just for the `builders.tar.gz` file that gets bundled with Now CLI (which will be going away very soon).